src/dymaptic.GeoBlazor.Core/Components/Layers/WCSLayer.cs

```diff
@@ -0,0 +1,147 @@
+﻿using dymaptic.GeoBlazor.Core.Components.Renderers;
+using dymaptic.GeoBlazor.Core.Exceptions;
+using Microsoft.AspNetCore.Components;
+using System.Security.Cryptography.X509Certificates;
+using System.Text.Json.Serialization;
+
+
+namespace dymaptic.GeoBlazor.Core.Components.Layers;
+
+/// <summary>
+///     WCS presents raster data from a OGC Web Coverage Service. Raster data are projected and rendered on the client-side.
+///     It supports versions 1.0.0, 1.1.0, 1.1.1, 1.1.2 and 2.0.1. For version 2.0.1, it supports servers that support 
+///     GEOTIFF coverage and implements the following extensions: Scaling, Interpolation, Range Subsetting, CRS, and KVP/Get.
+///     <a target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-WCSLayer.html">
+///         ArcGIS
+///         JS API
+///     </a>
+/// </summary>
+public class WCSLayer : Layer
+{
+    /// <inheritdoc />
+    [JsonPropertyName("type")]
+    public override string LayerType => "wcs";
+
+    /// <summary>
+    ///     Constructor for use as a razor component
+    /// </summary>
+    public WCSLayer()
+    {
+    }
+    /// <summary>
```

Comment:
there should be a space before these comments, and not one after them, for consistency.

Reply:
Corrected across all the class files.